### ISSUE TITLE
Backport #477 in the 2.0.x release branch

### ISSUE THREAD

**Issue by databus23** [quay/clair#630]
### Description of Problem / Feature Request

We just run into #441. The issue has been fixed with #477 almost a year ago in master.

Can we please backport this so that this issue is mitigated in production deployments? It should not introduce any incompatibilities or functional changes (besides that the container doesn't create thousands of zombies anymore.)

### Expected Outcome
No zombies

### Actual Outcome
Zombies!

### Environment

- Clair version/image: v2.0.4
- Clair client name/version: N/A
- Host OS: CoreOS
- Kernel (e.g. `uname -a`): 4.14.48-coreos-r2
- Kubernetes version (use `kubectl version`): 1.7.5
- Helm version (use `helm version`): 1.10
- Network/Firewall setup: N/A

**Comment by wdoekes** [comment]
If you're going to fix old builds, might as well remove this annoying "true" printed to stdout:

https://github.com/coreos/clair/blob/v2.0.4/api/v1/models.go#L246

:sunglasses:

**Comment by wdoekes** [comment]
.. and you'll want to cherry-pick 9e4a347ecdf15a503c5505ddb011b930c6b0e536.

**Comment by potzkovge** [comment]
While it was fixed in master(#477) (See: https://engineeringblog.yelp.com/2016/01/dumb-init-an-init-for-docker.html) You can fix it in docker-compose v2 with `init: true` as described here: https://docs.docker.com/compose/compose-file/compose-file-v2/#init
For v3 it is discussed here: https://github.com/docker/compose/issues/5049

So a working docker-compose v2 example would be:
```
version: "2"
services:
  clair:
    image: quay.io/coreos/clair:v2.0.2
    init: true
...
```

**Comment by hdonnay** [comment]
We’re declaring bug bankruptcy as part of the release process for a new major version of Clair. Please open a ticket in our [issue tracker](https://issues.redhat.com/projects/PROJQUAY/) if you feel this still needs to be addressed, and we'll triage as part of our v4 development process. Thanks!
